src/pages/home/Home.js

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import './Home.scss'
+import { Link } from 'react-router-dom'
+
+
+
+function Home() {
+
+    return (
+        <div className='header'>
+
+            <nav className="navbar navbar-expand-lg navbar-light">
+                <a class="navbar-brand" href="#"> <img src={require('../images/icons8-shopify-96.png')} alt='store icon' /></a>
```

Comment:
requiring an asset inline is not the perfect thing u can do, 
try importing the image above and then use it inside your template 

```js
import Icon from "../images/icons8-shopify-96.png"

// in the template
<img src={icon}/>
```